Resources/Prototypes/Entities/Clothing/Back/backpacks.yml

```diff
@@ -44,6 +46,8 @@
   components:
   - type: Sprite
     sprite: Clothing/Back/Backpacks/security.rsi
+  - type: Item
+    sprite: Clothing/Back/Backpacks/security.rsi
   - type: Clothing
     sprite: Clothing/Back/Backpacks/security.rsi
```

Comment:
The item & clothing component `sprite` fields default to using the sprite-component's `sprite` field. Its only required if the rsi differs from the sprite comp, or if it needs to override an inherited value (which is what your fix is doing).

In this case, the a better way of fixing this is just to remove the sprite field from those two components from the parent prototypes (ClothingBackpack & ClothingBackpackDuffel). That way no value is inherited and they will just use the default.

If you wanted to, you could then also remove all the clothing components from the inheritors, because they can all just use the default value.

Reply:
Updated.